### ISSUE TITLE
Feature request: Support Response file and Directive.

### ISSUE THREAD

**Issue by moh-hassan** [dotmake-build/command-line#17]
Thanks for this great tool.

When trying to use some built-in features in ```System.CommandLine``` like  [response files](https://learn.microsoft.com/en-us/dotnet/standard/commandline/syntax?source=recommendations#response-files)

example
```
app @file.rsp

```
i get error: 
``` Unrecognized command or argument '@file.rsp'. ```
 or 
```
app project list --account @account.txt
```
the option ```account``` should be read from file account.txt, but it use the value as is without resolving fileName
Or trying to use [directive](https://learn.microsoft.com/en-us/dotnet/standard/commandline/syntax?source=recommendations#directives) like:

```
app [parse] project list --verbose
```

I get error:
```
Unrecognized command or argument '[parse]'.
```

It is nice of these feature can be supported by dotmake-build

**Comment by moh-hassan** [comment]
I find by using ```CliSetting``` and set :
```
 CliSettings settings= new CliSettings
 {
     EnableDiagramDirective = true, //false by default
   
 };
 ```
 I can use the directive ```[diagram]``` instead of ```[parse]```
 
 It is better to unify the keyword used in System.Commandline and set ```EnableDiagramDirective =true``` by default.
 
 For respons file,  I find that the setting of  [ResponseFileTokenReplacer](https://github.com/dotmake-build/command-line/blob/6b6fc617ce9c7567e6ff4938dfedc128601d7650/src/DotMake.CommandLine/CliSettings.cs#L80)  is null (it is not configured).
 In system.CommandLine it is configured to use internal method as given below:
 ```
 ResponseFileTokenReplacer = StringExtensions.TryReadResponseFile;
 ```

**Comment by calacayir** [comment]
FYI, this is fixed with v1.8.8.

**Comment by moh-hassan** [comment]
Thanks for the new version.
